containers/conformance.md

```diff
@@ -173,29 +173,30 @@ In the runtime, the Zowe content is organized in this structure:
       +- /components
         +- /<component-id>
     +- /instance
-      +- instance.env or zowe.yaml
+      +- zowe.yaml
       +- /logs
-      +- /tmp
       +- /workspace
     +- /keystore
-      +- zowe-certificates.env
+      +- keystore.p12
```

Comment:
I don't think these files should be guaranteed to be here.
What we could say is "keystore content will be in the /keystore directory"
whether thats a p12, a cer, or both is really up to the users and would be specified in the zowe.yaml

Reply:
Agreed. changes wording.